### DISCUSSION TITLE
Modify how server.cfg is created

### DISCUSSION THREAD

**Discussion by crashzk** [seDirector/App#12] [Ideas]
Currently when we ask to create the server's server.cfg it creates based on this example: https://sedirector.net/server.cfg

It would be interesting to create it based on the server you are creating, take the following information and put it in **server.cfg**:
- hostname
- rcon_password
- sv_tags
- sv_password
- config fastdl

Something like this in **server.cfg**:
```
//HOSTNAME
hostname ""	// Server Name/Description

//RCON
rcon_password ""		// Set's remote control password

//PASSWORD
sv_password ""		// Password for users to connect to server

//TAGS
sv_tags ""

//FASTDL
sv_allowdownload "1"		// Allow clients to download files
sv_allowupload "1"		// Allow clients to upload customizations files
sv_downloadurl "http:// domain.com/gameServerFiles/"	// Location from which clients can download missing files
```
All those cvars, HOSTNAME, RCON, SV PASSWORD would be nice if it got automatically after you create the server.

Example, I would take the name of my server and already create based on it.
![image](https://user-images.githubusercontent.com/32937653/136469940-c60947e4-5ce9-4d21-9176-b5fcabc22111.png)

Only FastDL that would have to be manually configured. Or somehow after configuring FastDL on the server and clicking to enable HTTP Fast Download it updates the server.cfg and puts the information also based on what you configure in the Optimization tab.
![image](https://user-images.githubusercontent.com/32937653/136470234-f94abbdb-0393-468a-a957-46717ffa580b.png)

**NOTE: This configuration only for CSGO, I don't know if it would be good for all games, but I believe that by default the majority would be this one.**

**Comment by crashzk** [comment]
Sorry for the bad English, I don't know if I could understand everything.

**Comment by asheroto** [comment]
Yeah, I like this idea.  CFG editing is something I plan on implementing.
